{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "cveId": "CVE-2024-45346",
    "assignerOrgId": "b57733aa-7326-4f07-8e09-0be8e0df1909",
    "state": "PUBLISHED",
    "assignerShortName": "Xiaomi",
    "dateReserved": "2024-08-28T02:24:34.837Z",
    "datePublished": "2024-08-28T06:44:40.297Z",
    "dateUpdated": "2024-08-28T07:36:48.108Z"
  },
  "containers": {
    "cna": {
      "affected": [
        {
          "defaultStatus": "affected",
          "product": "GetApps application",
          "vendor": "Xiaomi",
          "versions": [
            {
              "changes": [
                {
                  "at": "30.6.0.2",
                  "status": "unaffected"
                }
              ],
              "lessThanOrEqual": "30.2.7.0",
              "status": "affected",
              "version": "GetApps application",
              "versionType": "custom"
            }
          ]
        }
      ],
      "datePublic": "2024-05-06T06:37:00.000Z",
      "descriptions": [
        {
          "lang": "en",
          "supportingMedia": [
            {
              "base64": false,
              "type": "text/html",
              "value": "<span style=\"background-color: rgb(245, 247, 249);\">A code execution vulnerability exists in the XiaomiGetApps application product. This vulnerability is caused by the verification logic being bypassed, and an attacker can exploit this vulnerability to execute malicious code.</span><br>"
            }
          ],
          "value": "A code execution vulnerability exists in the XiaomiGetApps application product. This vulnerability is caused by the verification logic being bypassed, and an attacker can exploit this vulnerability to execute malicious code."
        }
      ],
      "impacts": [
        {
          "descriptions": [
            {
              "lang": "en",
              "value": "GetApps application 30.2.7.0"
            }
          ]
        }
      ],
      "metrics": [
        {
          "cvssV3_1": {
            "attackComplexity": "LOW",
            "attackVector": "NETWORK",
            "availabilityImpact": "HIGH",
            "baseScore": 8.8,
            "baseSeverity": "HIGH",
            "confidentialityImpact": "HIGH",
            "integrityImpact": "HIGH",
            "privilegesRequired": "NONE",
            "scope": "UNCHANGED",
            "userInteraction": "REQUIRED",
            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
            "version": "3.1"
          },
          "format": "CVSS",
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ]
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "description": "A code execution vulnerability exists",
              "lang": "en"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "b57733aa-7326-4f07-8e09-0be8e0df1909",
        "shortName": "Xiaomi",
        "dateUpdated": "2024-08-28T07:36:48.108Z"
      },
      "references": [
        {
          "url": "https://trust.mi.com/misrc/bulletins/advisory?cveId=545"
        }
      ],
      "source": {
        "discovery": "UNKNOWN"
      },
      "title": "GetApps application has code execution vulnerability",
      "x_generator": {
        "engine": "Vulnogram 0.1.0-dev"
      }
    }
  }
}